java/src/main/native/src/ColumnVectorJni.cpp

```diff
@@ -563,6 +563,44 @@ JNIEXPORT jlong JNICALL Java_ai_rapids_cudf_ColumnVector_second(JNIEnv *env, jcl
   CATCH_STD(env, 0);
 }
 
+JNIEXPORT jlong JNICALL Java_ai_rapids_cudf_ColumnVector_weekDay(JNIEnv *env, jclass,
+                                                                jlong input_ptr) {
```

Comment:
Nit: Indentation of parameters is off here and in the other native methods that were added.